### PR TITLE
Delete obsolete code by using knip

### PR DESCRIPTION
**Pull request by apepper**
Closes #451.

### CODE REVIEW

File: .knip.jsonc [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    "src/Objs/**/*EditingConfig.ts",
+    "src/Objs/**/*EditingConfig.tsx",
+    "src/Objs/**/*LayoutComponent.tsx",
+    "src/Objs/**/*ObjClass.ts",
+
+    "src/Widgets/**/*Component.tsx",
+    "src/Widgets/**/*EditingConfig.ts",
+    "src/Widgets/**/*EditingConfig.tsx",
+    "src/Widgets/**/*LayoutComponent.tsx",
+    "src/Widgets/**/*WidgetClass.ts",
```
Comment (apepper) [inline .knip.jsonc:24]: I wasn't able to make `import.meta.glob` work with knip. That is why I whitelisted all those files here as well.

Comment (apepper) [reply .knip.jsonc:24]: I found https://github.com/webpro-nl/knip/issues/168, but it looks like a custom plugin is needed.

**Review by dcsaszar — APPROVED** [review]
Interesting stuff.

**Comment by apepper** [comment]
REVIEW ([0976250](https://github.com/Scrivito/scrivito-portal-app/pull/452/commits/09762509443aff4c4d702421888261526a27bc27))

**Pull request merged**